app/views/govuk_publishing_components/components/docs/modal_dialogue.yml

```diff
@@ -0,0 +1,71 @@
+name: Modal dialogue
+description: Shows only one section, with no other navigation options, until the user finishes the immediate task
+body: |
+  Use where the application has gotten into a state from which it shouldn’t or can’t proceed without input from the user or the state of the current page needs to be preserved.
```

Comment:
Could we change this to:

It can be used to highlight supplemental information without taking the user off the page, or used where the application has gotten into a state from which it shouldn’t or can’t proceed without input...

Just because this fits how I want to use it and I think it's an absolutely valid use case for it.